

function bindDataToException(resp) {

    alert("kk");
}